src/workers/postUpdated.ts

```diff
@@ -274,6 +275,41 @@ const updatePost = async ({
   return;
 };
 
+type GetSourcePrivacyProps = {
+  logger: FastifyBaseLogger;
+  entityManager: EntityManager;
+  data: Data;
+};
+const getSourcePrivacy = async ({
+  logger,
+  entityManager,
+  data,
+}: GetSourcePrivacyProps): Promise<boolean> => {
+  try {
+    let query = entityManager
+      .getRepository(Source)
+      .createQueryBuilder('source')
+      .select(['source.private']);
```

Comment:
Just `private` would do no?

Reply:
Without `source.` the tests would fail

Reply:
Oh I think it's because you define `source` in `createQueryBuilder` you can omit that as you already defined the repo.

Reply:
I think it would potentially crash with the join as well. I'll double check

Reply:
Couldn't get it to work without it. I'll just leave it there